src/web/modules/payment_links/detail.njk

```diff
@@ -0,0 +1,122 @@
+{% from "common/json.njk" import json %}
+{% extends "layout/layout.njk" %}
+
+{% block main %}
+  <span class="govuk-caption-m">{{ account.description or account.gateway_account_id }}</span>
+  <h1 class="govuk-heading-m">Payment link details</h1>
+
+  {% for message in messages %}
+    <div class="govuk-error-summary success-summary" role="alert">
+      <h2 class="govuk-error-summary__title">{{message}}</h2>
+    </div>
+  {% endfor %}
+
+  <dl class="govuk-summary-list">
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Name</dt>
+      <dd class="govuk-summary-list__value">{{ paymentLink.name }}</dd>
+      <dd class="govuk-summary-list__actions"></dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Description</dt>
+      <dd class="govuk-summary-list__value">
+        {% if paymentLink.description %}
+        {{ paymentLink.description }}
+        {% else %}
+        <i>(None set)</i>
+        {% endif %}
+      </dd>
+      <dd class="govuk-summary-list__actions"></dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">URL</dt>
+      <dd class="govuk-summary-list__value"><a class="govuk-link govuk-link--no-visited-state" href="{{ url }}">{{ url }}</a></dd>
+      <dd class="govuk-summary-list__actions"></dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Gateway account</dt>
+      <dd class="govuk-summary-list__value"><a class="govuk-link govuk-link--no-visited-state" href="/gateway_accounts/{{ paymentLink.gateway_account_id }}">{{ paymentLink.gateway_account_id }}</a></dd>
+      <dd class="govuk-summary-list__actions"></dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Language</dt>
+      <dd class="govuk-summary-list__value">{{ paymentLink.language }}</dd>
+      <dd class="govuk-summary-list__actions"></dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Amount</dt>
+      <dd class="govuk-summary-list__value">
+        {% if paymentLink.price %}
+        {{ paymentLink.price | currency }}
+        {% else %}
+        <i>(User can choose)</i>
+        {% endif %}
+      </dd>
+      <dd class="govuk-summary-list__actions"></dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Reference enabled</dt>
+      <dd class="govuk-summary-list__value">{{ paymentLink.reference_enabled | string | capitalize }}</dd>
+      <dd class="govuk-summary-list__actions"></dd>
+    </div>
+
+    {% if paymentLink.reference_enabled %}
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Reference label</dt>
+      <dd class="govuk-summary-list__value">
+        {% if paymentLink.reference_label %}
+        {{ paymentLink.reference_label }}
+        {% else %}
+        <i>(None set)</i>
```

Comment:
I suppose following the above pattern this could be `(User must choose)` or `(User entered)` or something

Reply:
oh yeah just noticed this is just on the `reference_label` and not on asking for reference from the user in particular -- that makes sense